### ISSUE TITLE
[Request] Title Scroll

### ISSUE THREAD

**Issue by derrgoo** [jurialmunkey/skin.eminence.2#109]
See pic below for example. Is it possible to have the title scroll. Can't always see episodes. 

![image](https://user-images.githubusercontent.com/63697905/147573503-857665a6-c1de-4e35-9d43-9ae8ec988298.jpeg)

**Comment by derrgoo** [comment]
Bump. Is this possible?

**Comment by jurialmunkey** [comment]
Yeah definitely possible. Just been caught up in Arctic Horizon 2 and TMDbHelper stuff lately but I'll look at some of these Eminence things soon. This one should be easy to do.

**Comment by derrgoo** [comment]
Great. Looking forward also to your update on horizon👍
